tests/spdx/writer/rdf/test_extracted_licensing_info_writer.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 spdx contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from rdflib import Graph, Literal, RDFS, URIRef
+from spdx.writer.rdf.writer_utils import spdx_namespace
+
+from spdx.writer.rdf.extracted_licensing_info_writer import add_extracted_licensing_info_to_graph
+from tests.spdx.fixtures import extracted_licensing_info_fixture
+
+
+def test_add_extracted_licensing_info_to_graph():
+    graph = Graph()
+    extracted_licensing_info = extracted_licensing_info_fixture()
+
+    add_extracted_licensing_info_to_graph(extracted_licensing_info, graph, URIRef("anyURI"), "anyURI")
```

Comment:
It seems to me that you don't include both parameters `URIRef("anyURI")` and `"anyURI"` in the assert statements. We should be able to include all parameters in the test, otherwise we don't know if they are processed correctly.

Reply:
This problem of untested parameters also applies to almost all other writers.

Reply:
Fixed, I also renamed the parameter as `anyURI` isn't really helpful  to understand the intent of the parameter.